pkg/scheduler/algorithmprovider/registry.go

```diff
@@ -121,6 +124,10 @@ func getDefaultConfig() *schedulerapi.Plugins {
 				{Name: noderesources.LeastAllocatedName, Weight: 1},
 				{Name: nodeaffinity.Name, Weight: 1},
 				{Name: nodepreferavoidpods.Name, Weight: 10000},
+				// Weight is doubled because:
+				// - This is a score coming from user preference.
+				// - It makes its signal comparable to NodeResourcesLeastAllocated.
```

Comment:
> It makes its signal comparable to NodeResourcesLeastAllocated.

is this comment accurate? am I misreading, or is NodeResourcesLeastAllocated weighted at 1 above?

Reply:
This is not a reference to ```NodeResourcesLeastAllocated```'s weight, but the score function itself which produces scores that significantly differentiates between nodes.